Build Break on Mac OS X

Build break on Mac OS X with latest commit. 
> cmake ..
...
> make
[ 23%] Built target jsoncpp_lib
[ 41%] Built target jsoncpp_object
[ 64%] Built target jsoncpp_static
[ 76%] Built target jsontestrunner_exe
[ 82%] Building CXX object src/test_lib_json/CMakeFiles/jsoncpp_test.dir/main.cpp.o
<>/jsoncpp/src/test_lib_json/main.cpp:356:12: error: implicit conversion changes signedness: 'int' to
      'Json::Value::ArrayIndex' (aka 'unsigned int') [-Werror,-Wsign-conversion]
  v.resize(n);
    ~~~~~~ ^
1 error generated.
make[2]: *** [src/test_lib_json/CMakeFiles/jsoncpp_test.dir/main.cpp.o] Error 1
make[1]: *** [src/test_lib_json/CMakeFiles/jsoncpp_test.dir/all] Error 2
make: *** [all] Error 2

--- a/src/test_lib_json/main.cpp
+++ b/src/test_lib_json/main.cpp
@@ -351,7 +351,7 @@ JSONTEST_FIXTURE_LOCAL(ValueTest, resizeArray) {
 }
 
 JSONTEST_FIXTURE_LOCAL(ValueTest, resizePopulatesAllMissingElements) {
-  int n = 10;
+  Json::ArrayIndex n = 10;
   Json::Value v;
   v.resize(n);
   JSONTEST_ASSERT_EQUAL(n, v.size());

